### ISSUE TITLE
Add instructions for zed editor in readme

### ISSUE THREAD

**Issue by linux-cultist** [nim-lang/langserver#239]
I tried a few times to configure nimlangserver with the zed editor but hasnt succeeded. Just some help in the readme would be much appriciated.

**Comment by foxoman** [comment]
Did you tried https://github.com/foxoman/zed-nim

**Comment by linux-cultist** [comment]
Yeah I did and it doesn't work... I was hoping for some tips on if I'm supposed to do something else than just install the language server. Because normally that is all it takes, but in this case I tried

- Installing nim language server from the OS packet repository. Didn't seem to do anything.
- Install nimble and try to install the nim language server using that, and add path to the system path. Didn't seem to do anything either.


So I'm not sure what to do. What steps do I need to take to get nim lsp support?
